README.md

```diff
@@ -25,12 +25,11 @@ export default {
 };
 </script>
 <template>
-  <section>
-    <fabra-connect :linkToken="ADD_GENERATED_LINK_TOKEN" /* :customTheme="customTheme" */>
-      <template slot="button" slot-scope="props">
-        <button @click="props.onClick">Open Fabra Connect</button>
-      </template>
-    </fabra-connect>
-  </section>
+  <div id="fabra-connect-container"></div>
+  <fabra-connect :linkToken="ADD_GENERATED_LINK_TOKEN" containerID="fabra-connect-container">
+    <template #button="props">
```

Comment:
`#button="props"` is the new syntax. `slot="button" slot-scope="props"` is from Vue2 and is deprecated in Vue3. I'm not sure where the merge-link library got this but I tried the old syntax and it doesn't work. :/ Here's the reference I found: 

https://stackoverflow.com/questions/57806610/how-to-use-slot-scope-correctly-within-vuejs-2-5-x
Old syntax:
https://v2.vuejs.org/v2/guide/components-slots.html#Scoped-Slots-with-the-slot-scope-Attribute

New syntax:
https://v2.vuejs.org/v2/guide/components-slots.html#Scoped-Slots-with-the-slot-scope-Attribute